testing/install-google-test.bash

```diff
@@ -0,0 +1,4 @@
+sudo apt install libgtest-dev build-essential cmake
+cd /usr/src/googletest
```

Comment:
am I missing a step? does libgtest-dev create this directory and install some unbuilt sources here?

Reply:
yep